docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -55,7 +55,7 @@ Version 2.0.30
   * `backup restore restore-disks`
   * `backup vault delete`
   * `backup vault show`
-* 'name' parameters now accept the name format as output from the show commands
+* `name` parameters now accept the name format output from backup show commands
```

Comment:
Still does not list the affected parameters. Please link a PR so that I can see what to document here.

Reply:
This was the change. It affects the `name` parameter but affects several commands.

https://github.com/Azure/azure-cli/commit/2ed6d9926e8914bc6ac9af3aec6084a7bb7b14c6#diff-03a39162274f27eea578f5b15294f416